.gitignore

```diff
@@ -4,6 +4,7 @@
 .config
 .yardoc
 Gemfile.lock
+gemfiles/*.lock
```

Comment:
Huh, I thought we were committing Gemfile.lock. Let's remove these ignores, add the lock file to git and remove the lock file when running on Travis like we do for Rails.